Pod/Classes/SEGAmplitudeIntegration.m

```diff
@@ -11,12 +11,21 @@ - (id)initWithSettings:(NSDictionary *)settings
         self.settings = settings;
```

Comment:
This method is not being tested all. 

All this logic should be in the second initializer (the initWithSettings and andAmplitude one).

Then this method should call that second initializer, using `Amplitude instance` as the parameter for `andAmplitude`.

Reply:
I made this change but the comment was on a line that didn't need to change